.editorconfig

```diff
@@ -0,0 +1,13 @@
+# https://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.md]
+max_line_length = off
+trim_trailing_whitespace = false
```

Comment:
why trailing whitespaces should be allowed in md files?

Reply:
As a matter fact, I have no idea, was curious about this myself. But I thought since I think all GH searches through your repos had this set, maybe you guys specifically somewhat agreed on this already.

OTOH, it could also be some kind of a boilerplate default that I'm not aware of?

Google pointed me to https://github.com/editorconfig/editorconfig-vscode/issues/153

> When I add 2 spaces to the end of a line to create a markdown line break and then save the file, the trailing spaces are removed.

So apparently trailing whitespace logic is part of MD.

Reply:
Interesting, thanks @lkraav!